tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -1333,7 +1337,7 @@ def _test_expansion_via_release_archive(
                 if name == "file.min.js" and not link_sourcemaps:
                     # Remove link to source map, add to header instead
                     content = content.replace(b"//@ sourceMappingURL=file.sourcemap.js", b"")
-                    entry["headers"] = {"Sourcemap": "/file.sourcemap.js"}
```

Comment:
I’m surprised a leading `/` does not work? Or was that fixed in the meantime?

Reply:
It does work, I just wanted to make it consistent, as `sourceMappingURL` doesnt have one.